api/src/list/dto/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './clone-list.dto';
```

Comment:
Do we need this file? You can export directly from the dto file itself no? `export class CloneListDto`, etc.